chapters/ch23.md

```diff
@@ -1,36 +1,35 @@
 # 23. Addressing Bias and Variance
--> 
-
+->
+Giải quyết Độ chệch và Phương sai
 
 Here is the simplest formula for addressing bias and variance issues:
 ->
-
+Đây là công thức đơn giản nhất để giải quyết các vấn đề độ chệch và phương sai:
 
 * If you have high avoidable bias, increase the size of your model (for example, increase the size of your neural network by adding layers/neurons).
 ->
-
+* Nếu bạn có độ chệch cao, hãy tăng kích thước mô hình của bạn (ví dụ: tăng kích thước của mạng neural bằng cách thêm các lớp/neurons)
```

Comment:
*layers* quy định trong glossary mình dịch là **tầng**.

Reply:
Chỗ này mình search in all files thấy chapter 1 dịch là lớp ._.
@tiepvupsu @aivivn @duythanhvn  mình thấy layer dịch lớp thì sẽ hợp lý hơn, mọi nguời thấy sao?

Reply:
> Chỗ này mình search in all files thấy chapter 1 dịch là lớp ._.
> @tiepvupsu @aivivn @duythanhvn mình thấy layer dịch lớp thì sẽ hợp lý hơn, mọi nguời thấy sao?

Vậy thì chỗ đó cần sửa lại là tầng. "Lớp" dễ nhầm với "class".